Sources/Harbor/Mock/HMock.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  HMock.swift
+//  Harbor
+//
+//  Created by Javier Manzo on 06/11/2024.
+//
+
+import Foundation
+
+@HRequestManagerActor
+public struct HMock: Sendable {
```

Comment:
There is no need to add sendable in structs if all properties are not modifiable